### ISSUE TITLE
Hunting dogs for detecting assassins. 

### ISSUE THREAD

**Issue by Russianstory** [UnofficialCrusaderPatch/UnofficialCrusaderPatch#906]
In another topic I talked about hiring an engineer with a hunter's dog. I suggest doing this:
The hunter's dogs are not fighting, so there is little strength. But this is a hunting dog, which means that it has an excellent sense of smell. Therefore, I propose to use this dog with a sense of smell, to DETECT ENEMY ASSASSINS. A type of dog with a good nose can smell assassins further than peaceful units and combat ones too. And so that the dog does not run by itself, then we will assign an engineer to it.

How do you like the idea?

**Comment by Monsterfisch** [comment]
just make dogcages detect assasins in a larger radius of about 10 tiles that way they could be placed at strategic locations to protect lets say the  granary more effectively

**Comment by Russianstory** [comment]
No cages with fighting dogs are suitable. There are two types of dogs - fighting and hunting. The hunting one is needed to detect the assassin, since he has a large range (due to his scent)

**Comment by Monsterfisch** [comment]
you need to think about it from a gameplay mechanic if you give this ability to hunter dogs they will only show if assassins are there at random spots on the map but with the dog cages it will give the player etc a chance to create safer areas considering the cages are already very expensive for what they do this would justify their price at least to some degree

**Comment by Russianstory** [comment]
But I would still divide the dogs into fighting and sentinel dogs.

**Comment by Monsterfisch** [comment]
why split it into smaller parts just combine it all into the dog cage which will give you control that you want to be guarded and gives it some basic protection without having to reinvent the wheel and add more complex and micromanage intensive mechanics that don't add enough to the game other than detecting assassins...

**Comment by Russianstory** [comment]
Well, I just want to use ordinary dogs. If the Caged Dogs can detect Assassins, then so can. I agree. But I thought different dogs were interesting too. I wrote about a hunting dog in another topic about city militias.

**Comment by Monsterfisch** [comment]
hunting dogs already fight as far as I know  and they likely also detect/show Assasins already but its unreliably 
as for the other topic you have made thats too ambitious to setup to consider (at least in my opinion)

**Comment by Russianstory** [comment]
So that's what I mean: Hunting Dog only detects Assassins. Combat (frenzied) only fights

**Comment by OverlordMatty** [comment]
Sure, that would increase the usability of the hunting post,
monsterfishs suggestion is fine as well. +1

**Comment by Russianstory** [comment]
Yes, I think any suggestion will do =) You just need to do something with NON-POPULAR buildings, units and objects. We need to increase their value and relevance, as happened with the pharmacist and spearmen MUCH. Their value has grown MUCH. =)

**Comment by Monsterfisch** [comment]
I would much rather like to see hunters generate food by taking cow carcases from tanners instead of being an underwhelming pseudo military unit btw i just tested it and seems like dog cages already are able to detect assasins from further away it would probably be good then to have the dogs respawn slowly considering the cost of the building.

**Comment by Russianstory** [comment]
I already wrote about the cow and the meat from it. I agree that would be great! But maybe for a start, just increase the number of cows to 7 in the pen? Three are constantly taken by the hunter - three are taken by the tanner, and one is left for the birth of offspring, so that the farm does not stand idle, but it would be possible to breed calves and milk milk

**Comment by Russianstory** [comment]
In this case, the hunter will no longer need a dog. It turns out an extra unit
